src/llmcompressor/entrypoints/README.md

```diff
@@ -0,0 +1,53 @@
+# LLM Compressor Entrypoints
+
+## Oneshot
+
+Model optimizations compress models while preserving accuracy. One-shot in LLM-Compressor supports faster inference on vLLM by applying post-training quantization (PTQ) or sparsification
+
+### PTQ
+PTQ is performed to reduce the precision of quantizable weights (e.g., linear layers) to a lower bit-width. Supported formats are:
+
+#### W4A16
+- Uses GPTQ to compress weights to 4 bits. 
+- Useful speed ups in low QPS regimes with more weight compression. 
+- Recommended for any GPUs types. 
+#### W8A8-INT8 
+- Uses channel-wise quantization to compress weights to 8 bits, and uses dynamic per-token quantization to compress activations to 8 bits. 
+- Useful for speed ups in high QPS regimes or offline serving on vLLM. 
+- Recommended for NVIDIA GPUs with compute capability <8.9 (Ampere, Turing, Volta, Pascal, or older). 
+#### W8A8-FP8
+- Uses channel-wise quantization to compress weights to 8 bits, and uses dynamic per-token quantization to compress activations to 8 bits. 
+- Useful for speed ups in high QPS regimes or offline serving on vLLM. 
+- Recommended for NVIDIA GPUs with compute capability >8.9 (Hopper and Ada Lovelace). 
+
+### Sparsification
+Sparsification reduces model complexity by pruning selected weight values to zero while retaining essential weights in a subset of parameters. Supported formats include:
+
+#### 2:4-Sparsity
+- Uses semi-structured sparsity, where for every four contiguous weights in a tensor, two are set to zero. 
+- Useful for efficiently computing sparse matrix multiplications using NVIDIA Sparse Tensor Cores. To preserve as much accuracy from the base model, usually a sparse-finetune step is added. 
+- Recommended for NVIDIA architecture from Amphere onwards, supported on GPUs with Sparse Tensor Cores
```

Comment:
I think this is great but I think we should do this as a follow-up so it's not blocking this PR after.